.github/workflows/bindings.yml

```diff
@@ -156,10 +156,13 @@ jobs:
         run: |
           cd interop/src/clients/InteropClient
           xcodebuild -scheme InteropClient -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 16' clean build install  DSTROOT=./Products
-          ./install_app.sh "iPhone 16"
-          ./grant_permissions.sh "iPhone 16"
+          echo "INTEROP_SIMULATOR_DEVICE=$(./create_simulator.sh)" >> GITHUB_ENV
       - name: run e2e interop test
         run: cargo run --bin interop
+      - name: delete simulator
+        run: |
```

Comment:
Suggest adding `if: always()` to this step so that the simulator gets cleaned up even if the test fails.